src/element/bounds.ts

```diff
@@ -186,7 +186,27 @@ export function getArrowPoints(
   return [x2, y2, x3, y3, x4, y4];
 }
 
-export function getCommonBounds(elements: readonly ExcalidrawElement[]) {
+export function getElementBounds(
+  element: ExcalidrawElement,
+): [number, number, number, number] {
+  const [x1, y1, x2, y2] = getElementAbsoluteCoords(element);
+  const angle = element.angle;
+  const cx = (x1 + x2) / 2;
+  const cy = (y1 + y2) / 2;
+  const [x11, y11] = rotate(x1, y1, cx, cy, angle);
+  const [x12, y12] = rotate(x1, y2, cx, cy, angle);
+  const [x22, y22] = rotate(x2, y2, cx, cy, angle);
+  const [x21, y21] = rotate(x2, y1, cx, cy, angle);
```

Comment:
```suggestion
  const cx = (x1 + x2) / 2;
  const cy = (y1 + y2) / 2;
  const [x11, y11] = rotate(x1, y1, cx, cy, element.angle);
  const [x12, y12] = rotate(x1, y2, cx, cy, element.angle);
  const [x22, y22] = rotate(x2, y2, cx, cy, element.angle);
  const [x21, y21] = rotate(x2, y1, cx, cy, element.angle);
```

Reply:
@lipis The intuition behind it was that it may or may not reduce a bundle size a bit.